packages/meditrak-server/src/routes/authenticate.js

```diff
@@ -70,6 +80,9 @@ const getAuthorizationObject = async ({ accessPolicy, refreshToken, user, permis
   if (permissionGroups) {
     userDetails.permissionGroups = permissionGroups;
   }
+  if (apiClientUser) {
+    userDetails.apiClient = apiClientUser.email;
```

Comment:
I'm a little confused about where this ends up getting used

Reply:
As discussed, nowhere, just a useful thing to see for devs when ensuring the user is logged in under the correct api client (also helps for tests 😉 )